inst/private/python_header.py

```diff
@@ -205,7 +206,8 @@ def octoutput(x, et):
             f.text = d2hex(x.real)
             f = ET.SubElement(a, "f")
             f.text = d2hex(x.imag)
-        elif isinstance(x, sp.compatibility.string_types):
+        elif isinstance(x, str) or \
+             (sys.version_info[0] <= 2 and isinstance(x, unicode)):
```

Comment:
As of 1.6, SymPy no longer itself supports Python 2, so there should be no need to use Python 2 compatibility idioms in this codebase, unless you are still supporting SymPy 1.5 and Python 2.

Reply:
thanks.  I kind of am, b/c my windows package relies on something (py2exe) that doesn't work on Py3.

Reply:
The py2exe on PyPI apparently supports Python 3 https://pypi.org/project/py2exe/

Reply:
Last I checked I was held up by https://github.com/manthey/pyexe/issues/25

Reply:
May I suggest switching to pyinstaller? It can also produce a single exe binary which packs all the needed dlls. It is also actively developed unlike pyexe.

Reply:
@cbm755: The Windows installer for Octave 6.x (built with MXE Octave) bundles a minimal version of python 3.8, sympy 1.4 and the symbolic package 2.9.0. So, it should no longer be necessary to provide a separate Windows installer for the symbolic package. (At least for newer versions of Octave.)
It would probably be possible to change to different versions of python or sympy in MXE Octave if that simplifies an update of the symbolic package.
If this is holding back getting the symbolic package to work with newer versions of sympy, you could probably just ignore backward compatibility with python2 (for Windows).